recipe/meta.yaml

```diff
@@ -14,89 +14,106 @@ source:
     - lib/pkgconfig/verilator.pc
 
 requirements:
   build:
     - {{ compiler('cxx') }}
-    - autoconf
-    - bison
-    - help2man
-    - make
+    - autoconf  # [not win]
+    - bison     # [not win]
+    - cmake     # [win]
+    - help2man  # [not win]
+    - make      # [not win]
     - perl
     - python
-    - sed
+    - sed       # [not win]
     - git # for patching
   host:
-    - flex  # need to link against it for target
+    - flex  # [not win]
+    - winflexbison  # [win]
     - zlib
   run:
-    - {{ compiler('cxx') }}  # [osx]
+    - {{ compiler('cxx') }}  # [osx or win]
     # The 4.034 branch was using this search and replace but it will break for osx-arm64
     #- {{ compiler('cxx') | replace('gxx_linux','gxx_impl_linux') | replace('_osx-64', '') }}
     # there must be a way to derive this from compiler('cxx') but this should work
     - gxx_impl_linux-64  # [linux64]
-    - {{ pin_compatible('flex') }}
-    - make
+    - {{ pin_compatible('flex') }}  # [not win]
+    - make  # [not win]
+    - cmake  # [win]
     - perl
     - python
 
-outputs:
-  - name: verilator
-    script: install.sh
-    requirements:
-      build:
-        - {{ compiler('cxx') }}
-        - make
-        - ripgrep
-        - perl # avoid BSD sed weirdness
-      host:
-        - zlib
-        - flex  # need to link against it for target
-      run:
-        - {{ compiler('cxx') }}  # [osx]
-        # there must be a way to derive this from compiler('cxx') but this should work
-        - gxx_impl_linux-64  # [linux64]
-        - {{ pin_compatible('flex') }}
-        - make
-        - perl
-        - python
-    test:
-      requires:
-        - pkgconfig
-        - cmake
-        - make
-        - python >=3
-        - graphviz
-        # only need C compiler because cmake bails if it doesn't have one.
-        # at least with the CMakeLists.txt in the examples
-        - {{ compiler('c') }}
-        # re-install the default compiler package with activation scripts
-        # because it is the easiest way to get cmake to work in the sanity checks
-        # for cmake.  However, normal makefile based tests do not need this and
-        # will work fine because verilated.mk builds in the path to the c++ compiler
-        - {{ compiler('cxx') }}
-      commands:
-        - pkg-config --print-provides "$PKG_NAME"
-        - verilator --version
-        - cp -r "$PREFIX"/share/verilator/examples .
-        - for d in examples/*; do echo $d; make -C $d; done
-  - name: verilator-debug
-    script: install.sh
-    requirements:
-      build:
-        - {{ compiler('cxx') }}
-      run:
-          # depending on the verilator package here breaks testing the package because conda-build
-          # can't install the verilator pakckage yet?  wtf?
-          #- {{ pin_subpackage('verilator', exact=True) }}
-    test:
-      commands:
-        - verilator_bin_dbg --version
+test:                                                   # [win]
```

Comment:
```suggestion
# we don't split the package on windows because it doesn't include the big debug build by default
test:                                                   # [win]
```